addon/services/hosted-service.js

```diff
@@ -82,6 +82,24 @@ export default Service.extend(serviceMixin, {
     }, portalOpts);
   },
 
+  /**
+   * Delete from the service definition
+   */
+  deleteFromDefinition (featureServiceUrl, definition, layerId, portalOpts) {
+    let adminUrl = featureServiceUrl.replace(/\/arcgis\/rest\/services\//i, '/arcgis/rest/admin/services/');
```

Comment:
feels like we could extract this into tested util fn to get an admin url from a feature service/layer's url used by at least this and `updateDefinition()`